Add changelog for version 1.0.2 (#13)
GeoWerkstatt/qgis-xtf-log-checker: xtflog_checker/metadata.txt

qgisMinimumVersion=3.0
description=Visualize errors in XTF-Log files
about=This QGIS plugin lets you display XTF-Log files created by the Ilivalidator (https://www.interlis.ch/downloads/ilivalidator). A checklist helps you keep track of your progress when addressing the errors.
version=1.0.2
author=GeoWerkstatt GmbH
email=support@geowerkstatt.ch

tags=interlis, ili, ilivalidator, xtf, transferfile, swiss, suisse, schweiz

> changelog=  1.0.2 Fixed bug where the "TechDetails" attribute was not read correctly.

Reviewer: PP: 
```suggestion
changelog=  1.0.2 The "TechDetails" attribute from the XTF-Log file was not read correctly.
```
